domains/certificates/Query.API/chart-overrides.yaml

```diff
@@ -1,4 +1,4 @@
-version: 0.0.19
+version: 0.0.20
```

Comment:
I would argue these changes merit a major version change instead of a patch.

Reply:
bumped in 18708ff